src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -279,7 +279,7 @@ public sealed partial class SocketsHttpHandler : System.Net.Http.HttpMessageHand
         public System.TimeSpan PooledConnectionLifetime { get { throw null; } set { } }
         public bool PreAuthenticate { get { throw null; } set { } }
         public System.Collections.Generic.IDictionary<string, object> Properties { get { throw null; } }
-        public System.Net.IWebProxy Proxy { get { throw null; } set { } }
+        public System.Net.IWebProxy? Proxy { get { throw null; } set { } }
         public System.TimeSpan ResponseDrainTimeout { get { throw null; } set { } }
         public System.Net.Security.SslClientAuthenticationOptions SslOptions { get { throw null; } set { } }
```

Comment:
We appear to allow null in the setter.

Reply:
yes it allows, and seems safe to have null, so i might better add [AllowNull]